CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Unreleased
 
+## Gesture
+- The experimental `IGesture` interface has changed. 
+	* The `Significance`, `Priority` and `PriotityAdjustment` have been merged into the single `GetPriority` function.
+	* `OnCapture` is changed to `OnCaptureChanged` and provides the previous capture state 
```

Comment:
Other multi-level lists use two spaces for indent, please use that in the future instead...